tests/unit/test.stickybits.js

```diff
@@ -127,13 +127,13 @@ test('stickybits .computeScrollOffsets interface', () => {
   // test instance setup
   expect(typeof instance).toBe('object')
   expect(typeof p).toBe('object')
-  const item = stickybit.computeScrollOffsets(instance)
+  stickybit.computeScrollOffsets(instance)
   // test .computeScrollOffsets interface
-  expect(typeof item).toBe('object')
-  expect(item.offset).toBe(0)
-  expect(item.stickyStart).toBe(0)
-  expect(item.stickyStop).toBe(0)
-  expect(item.state).toBe('default')
+  expect(typeof instance).toBe('object')
```

Comment:
Isn't this duplicate with the lines before?

Reply:
Yeah you’re right,  the typeof is a duplication...
Will remove this check ;)